ambry-account/src/main/resources/AccountSchema.ddl

```diff
@@ -26,7 +26,7 @@ CREATE TABLE IF NOT EXISTS Accounts
     INDEX lmtIndex (lastModifiedTime),
     INDEX statusIndex (status)
 )
-CHARACTER SET utf8 COLLATE utf8-bin;
```

Comment:
@Arun-LinkedIn I ran into collation errors when I ran the script on local Mysql db.  I couldn't find much info on utf8-bin,   Can we go with default collation for the char set?

Reply:
Seems like default collation for char set uf8 (utf8_general_ci) is case-insensitive and hence complains of unique key violation for case insensitive duplicate account or container names. This is differing from our code logic which seems to do case-sensitive comparison. May be we can update our code logic if there are no existing case insensitive duplicates in PROD? EI has one such case - two accounts with names 'MayFlower' and 'mayflower'.

Reply:
Since EI has such case, we probably need to maintain the case-sensitive comparison.

Reply:
Running the script on local mysql instance throws the following error:
ERROR 1273 (HY000): Unknown collation: 'utf8'

I'm not sure why it works on EI but we need an option that works consistently.

Reply:
Resolved by using utf8_bin.